openmrs/apps/reports/RADET/sql/pmtct_hei.sql

```diff
@@ -15,7 +15,7 @@ SELECT
 	getLastArvPickupDate(p.patient_id, "2010-01-01", "#endDate#") as "Last ARV Pickup Date / Dernière date de prise en charge des ARV",
 	getDurationMostRecentArvTreatment(p.patient_id, "2010-01-01", "#endDate#") as "Days of ARV Refill / Jours de recharge ARV",
 	getLocationOfArvRefill(p.patient_id, "2010-01-01", "#endDate#") as "Location of ARV refill / Emplacement de la recharge d'ARV",
-	"N/A" as "Differentiated ART delivery model at Last ARV refill ",
+	getDifferentiatedARTDeliveryModelAtLastRefill(p.patient_id,  "2010-01-01", "#endDate#") as "Differentiated ART delivery model at Last ARV refill ",
```

Comment:
@tmvumbi2 JP mentioned that there could be data as old as 2005. Should we make the default start date "2000-01-01"?